tests/test_explanation_dashboard.py

```diff
@@ -104,3 +104,18 @@ def test_interpret_dashboard(self, mimic_explainer):
                                     features=feature_names, classes=target_names)
         explanation = explainer.explain_global(x_test)
         show(explanation)
+
+    def test_visualize_explanation(self, mimic_explainer):
+        # Validate we can call the visualize method on the explanation
+        x_train, x_test, y_train, y_test, feature_names, target_names = create_cancer_data()
+        # Fit an SVM model
+        model = create_sklearn_svm_classifier(x_train, y_train)
+        explainer = mimic_explainer(model, x_train, LGBMExplainableModel,
+                                    features=feature_names, classes=target_names)
+        global_explanation = explainer.explain_global(x_test)
+        global_explanation.visualize()
```

Comment:
Can we assert on what we are expecting from the `visualize()` method? It seems you are expecting the call to not fail but the function seems to be returning something.

Reply:
the function just plots something

Reply:
maybe I can assert the type of plot or something